Dockerfile

```diff
@@ -7,17 +7,23 @@ ENV FAH_VERSION_MAJOR=7.5
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN useradd -ms /bin/bash folder
+RUN mkdir -p /etc/OpenCL/vendors && \
+    echo "libnvidia-opencl.so.1" > /etc/OpenCL/vendors/nvidia.icd
 
-RUN apt-get update && apt-get install --no-install-recommends -y \
+RUN apt-get update && apt-get install --no-install-recommends -y nvidia-opencl-dev ocl-icd-opencl-dev clinfo rsync wget vim file less \
         curl adduser bzip2 ca-certificates &&\
         curl -o /tmp/fah.deb https://download.foldingathome.org/releases/public/release/fahclient/debian-stable-64bit/v${FAH_VERSION_MAJOR}/fahclient_${FAH_VERSION_MINOR}_amd64.deb &&\
+        curl -o /tmp/fahcontrol.deb https://download.foldingathome.org/releases/public/release/fahcontrol/debian-testing-64bit/v7.4/fahcontrol_7.4.4-1_all.deb &&\
```

Comment:
We have existing containers for 7.4.4.

Reply:
I think I'll just remove this as the control doesn't install anyways (depends on gnome)

Reply:
I could never get the web interface to work unless I also installed fahcontrol, even though it had broken dependencies for the GUI it would still enable web. Have you had the same experience?

Reply:
For what it's worth, I've got the web interface working with the current version of the docker image.